### PR TITLE
add 'accessed' status to dashboard

### PR DESCRIPTION
**Pull request by ericcf**
[#100982574]

### CODE REVIEW

File: spec/models/concerns/status_spec.rb [hunk 1]
```diff
@@ -92,9 +100,11 @@ def initialize(study_day:, locale: "en", content_access_events: nil)
                 current_lesson = instance_double(Lesson,
                                                  guid: guid + "_abcd",
                                                  day_in_treatment: 1)
-                late_access = instance_double(ContentAccessEvent, late?: true)
-                allow(access_events).to receive(:where).with(lesson_id: lesson.id)
+                late_access = instance_double(SessionEvent, late?: true)
```
Comment (eschlange) [inline spec/models/concerns/status_spec.rb]: Why do you switch from using a ContentAccessEvent to a SessionEvent within the scope of these specs?

Comment (ericcf) [inline spec/models/concerns/status_spec.rb]: The original class, `ContentAccessEvent`, was unfortunately named. It actually represents a completion event. I need to go through and refactor that at some point. `SessionEvent` is a new class I added as a more generic representation. For now, it just represents accesses. Very confusing.

**Comment by eschlange** [comment]
Ah understood. Thanks for the clarification.

**Comment by eschlange** [comment]
Everything else looks good. Merging.

**Pull request merged**